### DISCUSSION TITLE
Export only selected Range

### DISCUSSION THREAD

**Discussion by arfx** [ggarra13/mrv2#215] [Ideas]
Hey, it would be awesome if you could add to the export a toggle to export only the selected Range (which should be set before via InPoint & OutPoint). It could have also a section where you can set in via Frame numbers or similar in the Export Setting.

**Comment by ggarra13** [comment]
By Export do you mean File->Save->Movie or Sequence?  This is already supported.  Mark an In and Out point with the 'i' and 'o' hotkeys *before* you call File->Save.  It will save only that segment instead of the whole movie.

**Comment by arfx** [comment]
Oh didn´t know sorry 🙏
